docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -208,6 +208,11 @@ Values containing `#` or `;` must be quoted using `` ` `` or `"""`.
 - `INTERNAL_TOKEN_URI`: **<empty>**: Instead of defining internal token in the configuration, this configuration option can be used to give Gitea a path to a file that contains the internal token (example value: `file:/etc/gitea/internal_token`)
 - `PASSWORD_HASH_ALGO`: **pbkdf2**: The hash algorithm to use \[pbkdf2, argon2, scrypt, bcrypt\].
 - `CSRF_COOKIE_HTTP_ONLY`: **true**: Set false to allow JavaScript to read CSRF cookie.
+- `PASSWORD_COMPLEXITY`: **lower,upper,digit,spec**: Set character types for password complexity. Lack or misconfiguration enable the complexity check requirement. Values of options: 
```

Comment:
`Lack or misconfiguration enable the complexity check requirement.` I don't understand this sentence. Can you explain it here so I can help you rewriting it?

Reply:
If no valid values are specified, the default option will be used.
default value is `PASSWORD_COMPLEXITY`: **lower,upper,digit,spec**